src/WebJobs.Script/Description/Binding/BindingMetadata.cs

```diff
@@ -80,6 +82,13 @@ public static BindingMetadata Create(JObject raw)
             string bindingDirectionValue = (string)raw["direction"];
             string connection = (string)raw["connection"];
             string bindingType = (string)raw["type"];
+
+            bool isProxy = false;
+            if(raw["isProxy"] != null && !bool.TryParse((string)raw["isProxy"], out isProxy))
```

Comment:
Aside from being the right type, if `isProxy` is a boolean in the JSON, this could be significantly simplified as you could eliminate the parsing and simply check the `Type` property here. This will also simplify our schemas.